### PR TITLE
Daml Finance: release notes adjustments

### PR DESCRIPTION
**Pull request by johan-da**

### CODE REVIEW

File: docs/3.0.0/docs/daml-finance/index.rst [hunk 1]
```diff
@@ -76,7 +75,98 @@ last release.
...
+while the ``Account`` also implements ``Lockable`` allowing to freeze an account, it’s not required.
+
+Additionally, the implementations of ``Transfer``, ``Split``, ``Merge``, and ``Debit`` have been
+adjusted to require unlocking before they can be used when in the locked state.
+
+New Instruments
+~~~~~~~~~~~~~~~
+
+Furthermore, we have broadened the library's functionality by introducing additional packages for
+new financial instruments, such as structured products, and floating rate instruments (both early
```
Comment (markus-da) [inline docs/3.0.0/docs/daml-finance/index.rst:153]:
```suggestion
new financial instruments, such as structured products, and multi-underlying asset swap instruments (both early
```

**Review by markus-da — COMMENTED** [review]
Left a minor suggestion regarding the new instruments

File: docs/3.0.0/docs/daml-finance/index.rst [hunk 2]
```diff
@@ -76,7 +75,98 @@ last release.
...
+interface (which the ``Holding`` now requires). That makes ``Lockable`` available for broader use;
+while the ``Account`` also implements ``Lockable`` allowing to freeze an account, it’s not required.
+
+Additionally, the implementations of ``Transfer``, ``Split``, ``Merge``, and ``Debit`` have been
+adjusted to require unlocking before they can be used when in the locked state.
+
+New Instruments
+~~~~~~~~~~~~~~~
+
+Furthermore, we have broadened the library's functionality by introducing additional packages for
```
Comment (markus-da) [inline docs/3.0.0/docs/daml-finance/index.rst:152]:
```suggestion
Furthermore, we have broadened the library's functionality by introducing
```

Comment (markus-da) [reply docs/3.0.0/docs/daml-finance/index.rst:152]: Only the structured product is in a new package, the new basket asset swap is in an existing package. I would maybe not mention package at all, but just focus on the new functionality offered in this release.

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 3]
```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
...
+package can be found as a sub-page :doc:`here <packages/index>`.
 
-Additionally, certain modifications have been implemented to ease future transitions to Daml 3.0 and
-the Canton Network.
+Enhanced Upgradeability, Extensibility, and Interoperability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section outlines the major changes and reasons behind them. Please also see the detailed
-changelog for each :doc:`package <packages/index>`.
+We have enhanced the core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces) to
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:88]:
```suggestion
The core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces) has been enhanced to
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 4]
```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
...
-Modifications to the core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces)
-have been made to improve extensibility and upgrade processes:
-
-#. The ``Account`` now uses a ``HoldingFactoryKey`` instead of a ``ContractId`` to reference its
-   ``Daml.Finance.Interface.Holding.Factory``. This allows for seamless ``HoldingFactory`` upgrades
-   without altering ``Account`` contract instances, as illustrated in our
+#. The ``Account`` now links to its ``Daml.Finance.Interface.Holding.Factory`` by key, a
+   ``HoldingFactoryKey``, instead of a ``ContractId``. This facilitates the upgrading of a
+   ``HoldingFactory`` without the need to modify existing ``Account`` contract instances. It also
+   enables a "lazy" upgrade approach for holdings, as detailed in our new
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:94]:
```suggestion
   enables a "lazy" upgrade approach for holdings, as detailed the
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 5]
```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
...
    :doc:`Holding Upgrade Tutorial <./tutorials/upgrade/holding>`.
 
-#. The ``Fungible`` interface no longer requires the ``Transferable`` interface. However, both
+#. In anticipation of the need for standardization when implementing composed workflows across
+   applications, we have introduced the notion of a ``HoldingStandard`` (as part of the
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:98]:
```suggestion
   applications, there is the notion of a ``HoldingStandard`` (as part of the
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 6]
```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
...
 
-#. The ``InstrumentKey`` now includes the ``HoldingStandard``, specifying the applicable holding
-   standard for an instrument. Settlement processes have been adjusted to only necessitate a
-   matching holding standard, allowing for implementation variations.
+Foreseeing future integration with Daml 3.0 and the Canton Network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Introducing a Locking Interface
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In order to ease future transitions to Daml 3.0 and the Canton Network we have shifted to
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:115]:
```suggestion
In order to ease future transitions to Daml 3.0 and the Canton Network, there is a shift to
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 7]
```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
...
-implementations without ``Lockable`` remain feasible.
+#. The `issuer : Party` of the ``InstrumentKey`` is now the single maintainer for the ``Instrument``
+   key.
+
+#. For ``Batch`` and ``Instruction``, the `requestors : Parties` field has been divided into a
+   single-maintainer `instructor : Party` for the ``Instruction`` key, alongside additional
+   signatories `consenters : Parties`. Corresponding changes have been made to the ``Batch`` and
+   ``Instruction`` views. In the ``Daml.Finance.Lifecycle.Rule.Claim`` implementation,
+   `providers : Parties` has been replaced with a single `provider : Party` (to facilitate assigning
+   the `provider` as a settlemetn `instructor : Party`).
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:126]:
```suggestion
   the `provider` as a settlement `instructor : Party`).
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 8]
```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
...
+
+#. The LedgerTime key has been completely removed as it was redundant.
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:128]:
```suggestion
#. The LedgerTime key has been completely removed, as it was redundant.
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 9]
```diff
@@ -131,32 +136,35 @@ facilitate the simultaneous archival of ``Account``, ``Instrument``, and ``Holdi
...
-   single-maintainer `instructor : Party` for the ``Instruction`` key, alongside additional
-   signatories `consenters : Parties`. Corresponding changes have been made to the ``Batch`` and
-   ``Instruction`` views. In the ``Daml.Finance.Lifecycle.Rule.Claim`` implementation,
-   `providers : Parties` has been replaced with a single `provider : Party` (to facilitate assigning
-   the `provider` as a settlemetn `instructor : Party`).
+New Instruments
+~~~~~~~~~~~~~~~
 
-#. The LedgerTime key has been completely removed as it was redundant.
+Furthermore, we have broadened the library's functionality by introducing new financial instruments,
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:152]:
```suggestion
The library's functionality has been broadened by introducing new financial instruments,
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 10]
```diff
@@ -131,32 +136,35 @@ facilitate the simultaneous archival of ``Account``, ``Instrument``, and ``Holdi
...
+such as structured products, and multi-underlying asset swap instruments (both early access).
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:153]:
```suggestion
such as structured products and multi-underlying asset swap instruments (both early access).
```

File: docs/2.8.0/docs/daml-finance/index.rst [hunk 11]
```diff
@@ -131,32 +136,35 @@ facilitate the simultaneous archival of ``Account``, ``Instrument``, and ``Holdi
...
+
+Usability Improvements
+~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, a large number (around 50 tickets) of smaller improvements addressing customer feedback
+have been made. These improvements range from the consistency of naming conventions (e.g. the type
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/index.rst:159]:
```suggestion
were made. These improvements range from the consistency of naming conventions (for example, the type
```

**Review by loiskilberg-da — COMMENTED** [review]
suggested some light edits

**Review by loiskilberg-da — APPROVED** [review]
thanks!

File: docs/2.8.0/docs/daml-finance/index.rst (hunk 5 above)
Comment (johan-da) [reply docs/2.8.0/docs/daml-finance/index.rst:98]: changed to:

In anticipation of the need for standardization when implementing composed workflows across
applications, the notion of a ``HoldingStandard`` was introduced (as part of the ``InstrumenKey``).

**Pull request merged**